.github/workflows/build-images.yml

```diff
@@ -84,10 +77,9 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: full_vertica_image
       with:
-        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}
+        default: docker.io/vertica/vertica-k8s-private:latest-master
         conditionals-with-values: |
           ${{ inputs.full_vertica_image != '' }} => ${{ inputs.full_vertica_image }}
-          ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind
 
     - name: Login to GitHub Container registry for non-PRs
       uses: docker/login-action@v2
```

Comment:
I can't make a comment on the exact line, but the next line when we check if we log into ghcr.io, we can remove the check for ''. We no longer default to a ghcr.io image. Something like this:
```
if: ${{ github.event_name != 'pull_request' && startsWith(inputs.full_vertica_image, 'ghcr.io') }}
```